### ISSUE TITLE
Service throws error with HTTP header Forwarded 

### ISSUE THREAD

**Issue by fw8** [CrunchyData/pg_featureserv#117]
I tried to run the pg_featureserv in google cloud run and the service always crashed with some cryptic error message

`runtime error: index out of range [0] with length 0`

Now I found out, that this has to do with a HTTP-Header that is added by Google when calling the service:

`Forwarded: for="109.90.42.114";proto=https`

To reproduce this you can start the container locally with:

`docker run --rm -e PGFS_SERVER_DEBUG=true -p 9000:9000 pramsey/pg_featureserv:latest --test`

Then in another terminal call the service with:

`curl -H 'Forwarded: for="109.90.42.114";proto=https'  "http://127.0.0.1:9000"`

Now you should see the error in the log:

```
time="2022-07-06T17:41:49Z" level=info msg="----  pg_featureserv - Version 1.3 ----------\n"
time="2022-07-06T17:41:49Z" level=info msg="Using config file: "
time="2022-07-06T17:41:49Z" level=info msg="Using database connection info from config file"
time="2022-07-06T17:41:49Z" level=info msg="Running in development mode"
time="2022-07-06T17:41:49Z" level=debug msg="Log level = DEBUG\n"
time="2022-07-06T17:41:49Z" level=debug msg="--- Configuration ---"
time="2022-07-06T17:41:49Z" level=debug msg="  BasemapUrl = *** NO URL PROVIDED ***"
time="2022-07-06T17:41:49Z" level=debug msg="  TableIncludes = []"
time="2022-07-06T17:41:49Z" level=debug msg="  TableExcludes = []"
time="2022-07-06T17:41:49Z" level=debug msg="  FunctionIncludes = [postgisftw]"
time="2022-07-06T17:41:49Z" level=info msg="Using Test Catalog data"
time="2022-07-06T17:41:49Z" level=info msg="Serving HTTP  at http://0.0.0.0:9000"
time="2022-07-06T17:41:49Z" level=info msg="CORS Allowed Origins: *\n"
time="2022-07-06T17:41:49Z" level=info msg="====  Service: pg-featureserv  ====\n"
time="2022-07-06T17:42:10Z" level=info msg="172.17.0.1:55450 GET /\n"
2022/07/06 17:42:10 http: panic serving 172.17.0.1:55450: runtime error: index out of range [0] with length 0
goroutine 34 [running]:
net/http.(*conn).serve.func1(0xc00043c000)
	/opt/hostedtoolcache/go/1.16.15/x64/src/net/http/server.go:1805 +0x153
panic(0xa93340, 0xc000466060)
	/opt/hostedtoolcache/go/1.16.15/x64/src/runtime/panic.go:971 +0x499
net/http.(*timeoutHandler).ServeHTTP(0xc0001cec80, 0xba8f58, 0xc00044c000, 0xc000442000)
	/opt/hostedtoolcache/go/1.16.15/x64/src/net/http/server.go:3302 +0x949
net/http.serverHandler.ServeHTTP(0xc0001341c0, 0xba8f58, 0xc00044c000, 0xc000442000)
	/opt/hostedtoolcache/go/1.16.15/x64/src/net/http/server.go:2868 +0xa3
net/http.(*conn).serve(0xc00043c000, 0xbaa840, 0xc000432040)
	/opt/hostedtoolcache/go/1.16.15/x64/src/net/http/server.go:1933 +0x8cd
created by net/http.(*Server).Serve
	/opt/hostedtoolcache/go/1.16.15/x64/src/net/http/server.go:2994 +0x39b
time="2022-07-06T17:42:10Z" level=debug msg="---- Request cancelled by client after 74.591µs"
```

It's the same problem that was reported by @aaronlaver in the issue [#107](https://github.com/CrunchyData/pg_featureserv/issues/107#issue-1188890334)

**Comment by dr-jts** [comment]
Looks like this is the same issue reported for `pg_tileserv` in https://github.com/CrunchyData/pg_tileserv/issues/74, and fixed in https://github.com/CrunchyData/pg_tileserv/commit/7438db1feff745e011efb599137eb5c2a50e4f91.

I will make the same fix here.

**Comment by dr-jts** [comment]
This should be fixed by https://github.com/CrunchyData/pg_featureserv/commit/f35fe8a8815c556c22833fe1c641c8607230f7cb

**Comment by fw8** [comment]
> This should be fixed by [f35fe8a](https://github.com/CrunchyData/pg_featureserv/commit/f35fe8a8815c556c22833fe1c641c8607230f7cb)

Yes, now it works also in google cloud run!

Thanks for the fixing this.
